custom_components/candy/binary_sensor.py

```diff
@@ -0,0 +1,325 @@
+        return UNIQUE_ID_WASHING_MACHINE_BINARY.format(self.config_id)
+
+    @property
+    def state(self) -> StateType:
+        status: WashingMachineStatus = self.coordinator.data
+        return str(status.machine_state)
+
+    @property
+    def icon(self) -> str:
+        return "mdi:washing-machine"
+
+    @property
+    def available(self):
+        status: WashingMachineStatus = self.coordinator.data
+        return status is not None
+
+    @property
+    def is_on(self):
+        status: WashingMachineStatus = self.coordinator.data
+        return status.machine_state not in [MachineState.IDLE] 
+
+
+class CandyTumbleDryerBinarySensor(CandyBaseBinarySensor):
+    def device_name(self) -> str:
+        return DEVICE_NAME_TUMBLE_DRYER
+
+    def suggested_area(self) -> str:
+        return SUGGESTED_AREA_BATHROOM
+
+    @property
+    def name(self) -> str:
+        return self.device_name()
+
+    @property
+    def unique_id(self) -> str:
+        return UNIQUE_ID_TUMBLE_DRYER_BINARY.format(self.config_id)
+
+
+    @property
+    def icon(self) -> str:
+        return "mdi:tumble-dryer"
+
+    @property
+    def available(self):
+        status: TumbleDryerStatus = self.coordinator.data
+        return status is not None
+
+    @property
+    def is_on(self):
+        status: TumbleDryerStatus = self.coordinator.data
+        return status.machine_state not in [MachineState.IDLE] 
+
+
+
+class CandyOvenBinarySensor(CandyBaseBinarySensor):
+    def device_name(self) -> str:
+        return DEVICE_NAME_OVEN
+
+    def suggested_area(self) -> str:
+        return SUGGESTED_AREA_KITCHEN
+
+    @property
+    def name(self) -> str:
+        return self.device_name()
+
+    @property
+    def unique_id(self) -> str:
+        return UNIQUE_ID_OVEN_BINARY.format(self.config_id)
+
+    @property
+    def icon(self) -> str:
+        return "mdi:stove"
+
+    @property
+    def available(self):
+        status: OvenStatus = self.coordinator.data
+        return status is not None
+
+    @property
+    def is_on(self):
+        status: OvenStatus = self.coordinator.data
+        return status.machine_state not in [OvenState.IDLE] 
+
+
+class CandyHobBinarySensor(CandyBaseBinarySensor):
+    def device_name(self) -> str:
+        return DEVICE_NAME_HOB
+
+    def suggested_area(self) -> str:
+        return SUGGESTED_AREA_KITCHEN
+
+    @property
+    def name(self) -> str:
+        return self.device_name()
+
+    @property
+    def unique_id(self) -> str:
+        return UNIQUE_ID_HOB_BINARY.format(self.config_id)
+
+    @property
+    def icon(self) -> str:
+        return "mdi:pot-steam"
+
+    @property
+    def available(self):
+        status: HobStatus = self.coordinator.data
+        return status is not None
+
+    @property
+    def is_on(self):
+        status: HobStatus = self.coordinator.data
+        return status.machine_state not in [HobState.IDLE] 
+
+class CandyDishwasherBinarySensor(CandyBaseBinarySensor):
```

Comment:
Same concern as with washing machines.

Reply:
This sensors just tell us if apliance work or not.
This information is useful in some UI integrations.

Example: I use  [minimalistic-area-card](https://github.com/junalmeida/homeassistant-minimalistic-area-card).  This card just show me white icon when appliance is on and gray icon when is of. But  homeassistant-minimalistic-area-card can use only binary sensor for this feature.